petl/util/timing.py

```diff
@@ -35,16 +37,49 @@ def progress(table, batchsize=1000, prefix="", out=sys.stderr):
     return ProgressView(table, batchsize, prefix, out)
 
 
+def log_progress(table, batchsize=1000, prefix="", out=None, level=logging.INFO):
```

Comment:
OK. Left it out so that it agreed with the progress() function, but this makes sense, too.